src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/PriceButton.tsx

```diff
@@ -49,20 +51,24 @@ const PriceButtonComp: React.FC<PriceButtonProps> = ({
       onClick={handleClick}
       priceAnnounced={priceAnnounced}
       disabled={isDisabled}
+      data-qa="price-button__trade-button"
+      data-qa-id={`direction-${direction.toLowerCase()}-${currencyPairSymbol.toLowerCase()}`}
```

Comment:
could potentially combine data-qa and data-qa-id
eg. `price-button__trade-button--${direction.toLowerCase()}-${currencyPairSymbol.toLowerCase()}`
but just a matter of preference..